io.github.hakuneko.HakuNeko.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.hakuneko.HakuNeko</id>
+	<launchable type="desktop-id">io.github.hakuneko.HakuNeko.desktop</launchable>>
+	<name>HakuNeko<name>
+	<project_license>Unlicense</project_license>
+	<developer_name>Ronny Wegener</developer_name>
+  	<summary>A cross-platform downloader for manga and anime from various websites</summary>
+  	<metadata_license>CC0-1.0</metadata_license>
+	<url type="homepage">https://github.com/manga-download/hakuneko</url>
+	<url type="bugtracker">https://github.com/manga-download/hakuneko/issues</url>
+	<url type="help">https://github.com/manga-download/hakuneko/wiki</url>
+	<description>
+		<p>HakuNeko is a cross-platform downloader for manga and anime from various websites. 
+		HakuNeko was made to help users downloading media for circumstances that require offline usage. 
+		The philosophy is ad-hoc consumption, get it when you going to read/watch it. 
+		It is not meant to be a mass downloader to stock up thousands of chapters that are just collected and will probably never be read.
```

Comment:
Unclosed `<p>` tag.

Reply:
Ok, I closed the <p> tag in the metainfo.